common/components/ConfirmationModalTemplate/ConfirmationModalTemplate.tsx

```diff
@@ -114,29 +114,23 @@ class ConfirmationModalTemplateClass extends React.Component<Props, State> {
     ];
 
     return (
-      <div className="ConfModalWrap">
-        <Modal
-          title="Confirm Your Transaction"
-          buttons={buttons}
-          handleClose={onClose}
-          disableButtons={transactionBroadcasting}
-          isOpen={true}
-        >
-          <div className="ConfModal">
-            {transactionBroadcasting ? (
-              <div className="ConfModal-loading">
-                <Spinner size="x5" />
-              </div>
-            ) : (
-              <div>
-                {this.props.summary}
-                {this.props.details}
-                <div className="ConfModal-confirm">{translate('SENDModal_Content_3')}</div>
-              </div>
-            )}
-          </div>
-        </Modal>
-      </div>
+      <Modal
+        title="Confirm Transaction"
+        buttons={buttons}
+        handleClose={onClose}
+        disableButtons={transactionBroadcasting}
+        isOpen={isOpen}
+      >
+        {transactionBroadcasting ? (
+          <React.Fragment>
+            <Spinner size="x5" />
+          </React.Fragment>
+        ) : (
+          <React.Fragment>
+            <Body />
```

Comment:
This breaks #940, a custom body should be able to be supplied for different transaction use-cases (ENS, contract interact, contract deploy could all have different displays to the user that make more sense in that context. Such as if the user is interacting with a contract we should be displaying what function they're calling in the confirmation modal. Or if it's ENS, to display ENS specific data in the confirmation modal.)

Reply:
Ideally, the `Body` component should be placed in a folder where `ConfirmationModal` implements `ConfirmationModalTemplate`, see https://github.com/MyCryptoHQ/MyCrypto/pull/940/files#diff-2bac20a9e428f854eb8c93128d97e2ea